.github/workflows/baseline-profile.yml

```diff
@@ -0,0 +1,85 @@
+name: Baseline profile generation
+
+on:
+  # every day at 00:43
+  schedule:
```

Comment:
instead of schedule please set only run on push of release branch, and auto-push commit in same branch

Reply:
I don't have a release branch. If you have one, then yes that would work.